libs/extensions/angular/.storybook/styles.css

```
@@ -0,0 +1,3 @@
+.canvas {
+    background-color: var(--kirby-background-color)!important;
```

Not important 😉 (pun intented) and not technically required, but consider adding a space before the `!important` flag. I think it makes it a bit more legible.

```suggestion
    background-color: var(--kirby-background-color) !important;
```